packages/flutter/lib/src/cupertino/page.dart

```diff
@@ -450,10 +450,13 @@ class _CupertinoBackGestureDetectorState extends State<_CupertinoBackGestureDete
       fit: StackFit.passthrough,
       children: <Widget>[
         widget.child,
-        new Listener(
-          onPointerDown: _handlePointerDown,
-          behavior: HitTestBehavior.translucent,
-          child: new SizedBox(width: _kBackGestureWidth),
+        new Align(
```

Comment:
Ah works. Thanks. 

I think it would be good to add some more sizing behaviour documentations on Positioned or ParentDataWidget.